### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by zysjimmy**
1. Fix the test case
2. Unify the value of field "categorization" in the database

### CODE REVIEW

File: sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java [hunk 1]
```diff
@@ -197,8 +197,6 @@ public void bookingTest() throws Exception {
 		reply = bookingTS.process(testerId, "I would like to book tour 2D001");
 		reply = bookingTS.process(testerId, "Yes.");
 		assertThat(reply).isEqualTo("On which date you are going? (in DD/MM format)");
-		reply = bookingTS.process(testerId, "21/11");
```
Comment (Jekyll-SONG) [inline sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java]: why delete this one?

Comment (zysjimmy) [reply sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java]: My code has changed thus this line is not applicable any more.

Comment (Jekyll-SONG) [reply sample-spring-boot-kitchensink/src/test/java/com/example/bot/spring/KitchenSinkTester.java]: ok, apporved

File: sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/DBEngine.java [hunk 2]
```diff
@@ -78,8 +78,8 @@ protected Connection getConnection() throws URISyntaxException, SQLException {
 		String password = dbUri.getUserInfo().split(":")[1];
 		String dbUrl = "jdbc:postgresql://" + dbUri.getHost() + ':' + dbUri.getPort() + dbUri.getPath() +  "?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory";
 
-		log.info("Username: {} Password: {}", username, password);
-		log.info ("dbUrl: {}", dbUrl);
+		//log.info("Username: {} Password: {}", username, password);
```
Comment (QIYUANYUAN) [inline sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/DBEngine.java]: why we don't need the log

Comment (zysjimmy) [reply sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/DBEngine.java]: It always print out the username and password of the database in the command lines. 
This is not appropriate regarding security issues. 
Also it's annoying that it always occurs in the command lines.

Comment (QIYUANYUAN) [reply sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/DBEngine.java]: well, that makes sense

**Pull request merged**